dasharo/project_config.yaml

```diff
@@ -0,0 +1,27 @@
+# Dasharo project configuration data
+
```

Comment:
dasharo:platform can be added as here: https://github.com/Dasharo/coreboot/blob/raptor-cs_talos-2/develop/dasharo/project_config.yaml#L3

Reply:
fixed the name